### ISSUE TITLE
Problems after update to 4.1.0

### ISSUE THREAD

**Issue by 1ctw** [Clinical-Genomics/scout#996]
The "Comments" in cases are changed to "None" after the update. Not all comments but there are both old and new ones that are affected. 

See for example:
https://scout.scilifelab.se/cust003/18104
https://scout.scilifelab.se/cust003/18028

**Comment by KickiLagerstedt** [comment]
I see this too!!!
https://scout.scilifelab.se/cust002/F0005922

**Comment by KickiLagerstedt** [comment]
Verkar vara kopia av "activity": när ngt gjorts med provet så får man kommentar "none"

**Comment by dnil** [comment]
On my end, it looks like other events than "comments" are rendered as comments.

**Comment by northwestwitch** [comment]
Yeah, something went wrong here. We'll try to fix it as soon as possible!
